packetbeat/protos/mysql/mysql.go

```diff
@@ -58,6 +62,10 @@ type mysqlMessage struct {
 	cmdlineTuple *common.CmdlineTuple
 	raw          []byte
 	notes        []string
+
+	statementID       int
+	numberOfParameter int
+	param             string
```

Comment:
why not `param []string`? The execute statement contains an array of parameters.

Reply:
yes, the execute statement contains an array of parameters.
actually I joint the array of parameters with “#” for easy in my circumstance

Reply:
What's the reason to separate paramters by '#'? Can the '#' symbol be part of a string parameter, getting you a false-positive separator char?

hm... I'd rather see it being `[]string`. Or introduce a setting to allow for both styles.

Reply:
`[]string` better